Vungle/VungleRouter.m

```diff
@@ -264,15 +273,24 @@ - (void)requestBannerMrecAdWithPlacementID:(NSString *)placementID delegate:(id<
         [tempDic setObject:delegate forKey:kVungleBannerDelegateKey];
         [tempDic setObject:[NSNumber numberWithInt:BannerRouterDelegateStateRequesting] forKey:kVungleBannerDelegateStateKey];
         [self.bannerDelegates addObject:tempDic];
-
+        
```

Comment:
Empty space

Reply:
If you need the extra line here - just be sure that the change (likely caused by the standard indent) is removed.

Reply:
Got it, I will remove the change. 😀